nwbwidgets/test/test_ophys.py

```diff
@@ -132,10 +122,30 @@ def setUp(self):
         self.df_over_f = DfOverF(rrs)
 
     def test_show_two_photon_series(self):
-        assert isinstance(
-            TwoPhotonSeriesWidget(self.image_series2, default_neurodata_vis_spec),
-            widgets.Widget,
+        self.image_series2 = TwoPhotonSeries(
```

Comment:
If you are going to move the construction down here then it should not be an attribute, since it is never passed between methods